content/_index.md

```diff
@@ -103,17 +103,17 @@ sections:
     background: gray
     title: Latest Posts
 seo:
-  title: Stackbit Azimuth Theme
-  description: The preview of the Azimuth theme
+  title: TARDIS-SN
+  description: Website for TARDIS-SN 
```

Comment:
This is not how you describe something. Suppose you shared Numfocus website on twitter, it shouldn't say Website for NumFOCUS, right? :sweat_smile:

It instead describes what NumFOCUS is. Do the same for TARDIS for all 3 descriptions - you can use our twitter bio (https://twitter.com/tardis_sn)